sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/_tests_/cas-supplier-integration.scheduler.e2e-spec.ts

```diff
@@ -366,6 +368,115 @@ describe(describeProcessorRootTest(QueueNames.CASSupplierIntegration), () => {
     },
   );
 
+  it("Should create a new site and update the student CAS supplier when an active CAS supplier exists with no match addresses.", async () => {
+    // Arrange
+    const referenceDate = new Date();
+    const savedCASSupplier = await saveFakeCASSupplier(db);
+    // Configure CAS mock to return a result for the GetSupplier
+    // with the same supplier number, supplier name and address line 1
+    // from the saved CAS supplier but a different postal code.
+    // and a successful result for the CreateSupplierAndSite.
+    casServiceMock.getSupplierInfoFromCAS = jest.fn(() =>
+      Promise.resolve(
+        createFakeCASSupplierResponse({
+          initialValues: {
+            supplierNumber: savedCASSupplier.supplierNumber,
+            addressLine1: savedCASSupplier.supplierAddress.addressLine1,
+            postalCode: "V1V1V1",
```

Comment:
Please add a comment here to mention that postal code is added to mismatch the address.